src/main/gui/create.ts

```diff
@@ -55,7 +55,9 @@ export const createGuiApp = (args: OpenArguments) => {
         app.disableHardwareAcceleration();
     }
 
-    const hasInstanceLock = app.requestSingleInstanceLock();
+    const isAllowMultipleInstances = settingStorage.getItem('allow-multiple-instances') === 'true';
+
+    const hasInstanceLock = app.requestSingleInstanceLock() || isAllowMultipleInstances;
```

Comment:
`app.requestSingleInstanceLock()` likely has numerous side effects, so let's swap those 2 conditions.
```suggestion
    const hasInstanceLock = isAllowMultipleInstances || app.requestSingleInstanceLock();
```